dissect/target/plugins/apps/webserver/apache.py

```diff
@@ -547,7 +553,17 @@ def infer_access_log_format(line: str) -> LogFormat | None:
         if line[-1] == '"':
             # ends with a quotation mark but does not contain a response time, meaning there is only a user agent
             return LOG_FORMAT_ACCESS_COMBINED
-        if line[-1].isdigit():
+        if line[-1].isdigit() or line[-1] == "-":
+            # ends with a digit or '-' indicating response size in bytes
             return LOG_FORMAT_ACCESS_COMMON
 
         return None
+
+
+def clean_value(value: str | None) -> str | None:
+    """Clean the given value by replacing empty strings and ``"-"`` with ``None``."""
+
+    if not value or value == "-":
```

Comment:
```suggestion
    if value in ("-", ""):
```
Maybe a bit clearer.